thoughtbot-about.md

```diff
@@ -0,0 +1,35 @@
+# thoughtbot.social
+
+## Code of Conduct
+
+In order to foster an inclusive, kind, harassment-free, and cooperative community, thoughtbot enforces this code of conduct on this social network.
+
+### Summary
+
+Harassment in code and discussion or violation of physical boundaries is completely unacceptable anywhere in thoughtbot’s social network, project codebases, issue trackers, chatrooms, mailing lists, meetups, and other events. The core team uses their discresion to choose whether to warn violators. Repeat violations will result in being blocked or banned by the core team at or before the 3rd violation.
+
+Existing employees are expected to live up to the same standards and will be handled by our internal processes.
+
+### In detail
+
+Harassment includes offensive verbal comments related to gender identity, gender expression, sexual orientation, disability, physical appearance, body size, race, religion, sexual images, deliberate intimidation, stalking, sustained disruption, and unwelcome sexual attention.
+
+Individuals asked to stop any harassing behavior are expected to comply immediately.
+
+Existing employees are also subject to the anti-harassment policy. Violation will be handled internally in our usual manner.
+
+If anyone engages in harassing behavior, including maintainers, we may take appropriate action, up to and including warning the offender, escalation to their admin, and defederation.
+
+If you are being harassed, notice that someone else is being harassed, or have any other concerns, please report them immediately.
```

Comment:
At some point we might want to add instructions on how to report.

Reply:
Good idea. Maybe this:

```suggestion
If you are being harassed, notice that someone else is being harassed, or have any other concerns, please report them immediately in your Mastodon app. You can do this by clicking `…` on the post, then "Report". All of these reports go to us; you may also choose to send an anonymous report to the other instance's moderation team.

If you report a teammate to us, we will handle it with discretion as [documented in our Handbook](https://github.com/thoughtbot/handbook/blob/ed1b747fc285c8da343c84b6d2b20db2cbf5e0fb/policies/conduct.md#reporting-and-discipline) (internal link).
```

Reply:
I think a better link is probably https://hub.thoughtbot.com/handbook/policies/conduct.md#reporting-and-discipline

Reply:
Thanks, updated with the better link.